packages/bar/src/types.ts

```diff
@@ -259,6 +258,9 @@ export type BarCommonProps<RawDatum> = {
     renderWrapper?: boolean
 
     initialHiddenIds: readonly (string | number)[]
+
+    enableTotals?: boolean
```

Comment:
No need for `?` for values having a default, the top components are using `Partial`, and underlying components are going to inherit the default value, so this won't be optional anymore.